src/views/preview/comment/compose-comment.jsx

```diff
@@ -183,7 +183,7 @@ ComposeComment.propTypes = {
     onAddComment: PropTypes.func,
     onCancel: PropTypes.func,
     parentId: PropTypes.number,
-    projectId: PropTypes.number,
+    projectId: PropTypes.string,
```

Comment:
should maybe be `PropTypes.oneOfType([PropTypes.string, PropTypes.number])`

Reply:
I think it is always a string, judging from the other places that have a propType for it (presentation.jsx)